packages/admin/cms-admin/src/dam/mediaAlternatives/MediaAlternativesGrid.tsx

```diff
@@ -76,6 +77,8 @@ const deleteDamMediaAlternativeMutation = gql`
     }
 `;
 
+export const mediaAlternativesGridRefetchQueries = [damMediaAlternativesQuery, VideoPreviewCaptionsQueryName];
```

Comment:
Why did you decide to use the query name for one item and the actual query for the other?